src/brs/Burst.java

```diff
@@ -305,12 +317,15 @@ public static void shutdown(boolean ignoreDBShutdown) {
       api.shutdown();
     if (apiV2Server != null)
       apiV2Server.shutdownNow();
-    Peers.shutdown(threadPool);
-    threadPool.shutdown();
+    if (threadPool != null) {
```

Comment:
Avoiding null pointer exceptions when you have something wrong on the conf files and stuff ends up unitialized.